### PR TITLE
Build fixes

### PR DESCRIPTION
**Pull request by rylev**
* Make Clippy happy
* Remove the smoke test
* Remove factors specific CI

### CODE REVIEW

File: crates/componentize/src/module_info.rs
Comment (lann) [inline crates/componentize/src/module_info.rs:1]: I'm having trouble getting my local clippy to be angry about this :thinking:

Comment (rylev) [reply crates/componentize/src/module_info.rs:1]: Which version of Rust are you running?

Comment (rylev) [reply crates/componentize/src/module_info.rs:1]: In any case, I think the `||` is nicer 🤷

**Review by lann — APPROVED** [review]

File: crates/componentize/src/module_info.rs
Comment (lann) [reply crates/componentize/src/module_info.rs:1]: 1.80

**Pull request merged**